docs/src/modules/java/pages/service-to-service.adoc

```diff
@@ -0,0 +1,111 @@
+include::ROOT:partial$include.adoc[]
+
+= Service to Service Eventing
+:page-supergroup-java-scala: Language
+
+Kalix provides broker less at-least-once event delivery across Kalix services through the Service to Service eventing.
```

Comment:
is it "brokerless" or "broker less"?

Reply:
brokerless

Reply:
CI spellcheck disagrees :/ 

But I think you are right, so I'll see about some allow-list for that spellcheck/lang tool.